coursemapper-kg/recommendation/app/services/course_materials/recommendation/recommender.py

```
@@ -206,295 +169,147 @@
+            total_time = time.time() - start_time
+            logger.info(f"Algorithm Model 1: Execution time {str(total_time)}")
+            return data
+        
         # If Model 2
-        elif recommendation_type == RecommendationType.DYNAMIC_DOCUMENT_BASED:
-            # Transform embedding to tensor
-            user_embedding_array = user_embedding.split(",")
-            user_embedding_array = [float(i) for i in user_embedding_array]
-            user_embedding_array = np.array(user_embedding_array)
-            user_tensor = torch.from_numpy(user_embedding_array).float()
+        elif recommendation_type == RecommendationType.PKG_BASED_DOCUMENT_VARIANT:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)